protocols/index.md

```diff
@@ -21,6 +21,7 @@ onedrive
 sharepoint
 box
 files.com
+sftptogo
```

Comment:
In general we will not want to have a single provider in the top level protocol list. Exceptions apply.